task.go

```diff
@@ -330,7 +330,7 @@ func (config TaskConfig) validateInputContainsNames() []string {
 type TaskRunConfig struct {
 	Path string   `json:"path" yaml:"path"`
 	Args []string `json:"args,omitempty" yaml:"args"`
-	Dir  string   `json:"dir",omitempty" yaml:"dir"`
```

Comment:
I'll track this as a separate bug and see if it actually affected anything. Thanks!